crates/tauri-bundler/src/bundle/windows/msi/wix.rs

```diff
@@ -540,8 +540,8 @@ pub fn build_wix_app_installer(
   let shortcut_guid = generate_package_guid(settings).to_string();
   data.insert("shortcut_guid", to_json(shortcut_guid.as_str()));
 
-  let app_exe_name = settings.main_binary_name().to_string();
-  data.insert("app_exe_name", to_json(app_exe_name));
+  let main_binary_name = settings.main_binary_name().to_string();
+  data.insert("main_binary_name", to_json(&main_binary_name));
```

Comment:
also unused, but i guess that's fine for custom templates? if so we should comment here

Reply:
yeah it is unused but I think it should stay for custom templates, will add a comment